realsense2_camera/src/profile_manager.cpp

```diff
@@ -333,12 +406,69 @@ void VideoProfilesManager::registerVideoSensorParams()
                     }
                     else
                     {
-                        ROS_ERROR_STREAM("Given value, " << parameter.get_value<std::string>() << " is invalid. Set ROS param back to: " << crnt_profile_str.str());
+                        ROS_ERROR_STREAM("Given value, " << parameter.get_value<std::string>() << " is invalid. "
+                                                    << "Run 'ros2 param describe <your_node_name> " << parameter.get_name() 
+                                                    << "' to get the list of supported profiles. "
+                                                    << "Setting ROS param back to: " << crnt_profile_str.str());
                     }
                     _params.getParameters()->queueSetRosValue(parameter.get_name(), crnt_profile_str.str());
                 }
             }, crnt_descriptor);
     _parameters_names.push_back(param_name);
+
+    for (auto sip : sips)
+    {
+        std::string param_name(_module_name + ".profile." + STREAM_NAME(sip) + "_stream_format");
```

Comment:
can we work with a shorter name ?
like
depth_module.infra1_format ?
I found that long names are not very easy to understand, and can easily used in a wrong way.